docs/RELEASE.md

```diff
@@ -0,0 +1,128 @@
+# Releasing the Operator
+
+This document walks through the release process.
+
+## Konflux
+
+Konflux is used for releasing our images. We have two applications for our operator: kueue and kueue-operator.
+
+To release, it is necessary to understand ReleasePlanAdmissions (RPA), EnterpriseContractPolicy (ECP), ReleasePlans (RP), Releases and Snapshots.
+You can find these definitions in konflux-release-data in our internal gitlab.
+
+A VPN is required for these links but you can find the definitions [here](https://gitlab.cee.redhat.com/releng/konflux-release-data).
```

Comment:
instead of pointing the link to the repository it might be worth including the direct links of reach resource, like https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/kueue-operator/kueue-operator-1-0-prod.yaml?ref_type=heads